packages/automerge-repo-sync-server/package.json

```diff
@@ -5,7 +5,7 @@
   "license": "MIT",
   "type": "module",
   "scripts": {
-    "start": "node ./src/index.js"
+    "start": "set DEBUG=* && node ./src/index.js"
```

Comment:
whoops

Reply:
would you mind just removing this from the PR? i'm happy to merge if so. given your broadcastchannel note we should probably also do something more sensible to reduce "echo" but i don't want to block on that.

Reply:
Sure, fixed. Making an issue for broadcastchannel/

Reply:
Sure. Created #50